crowdsec-docs/docs/expr/cti_helpers.md

```diff
@@ -0,0 +1,68 @@
+---
+id: cti_helpers
+title: CTI helpers
+sidebar_position: 4
+---
+
+## CTI Helpers
+
+CTI Helper allows to query Crowdsec's CTI API to enhance the engine capabilities.
+It requires [creating a CTI API Key in the console, as described here](/docs/next/cti_api/getting_started).
+
+The CTI API Key must be present in the `api.cti` section of the configuration file:
+
+```yaml
+api:
+  cti:
+    #The API key you got from the console
+    key: <API CTI KEY>
+    #How long should CTI lookups be kept in cache
+    cache_timeout: 60s
```

Comment:
How often does an IP update realistically? We should put a sensible default if this is then class as resolved

Reply:
yes you're right, I will bump that number